### PR TITLE
*: Update roadmap.md

### LINKED ISSUE #42178: update TiDB roadmap
## Enhancement
minor change to the description of database entryption

### PR DESCRIPTION
**Pull request by yiwen92**
minor change on security feature

### What problem does this PR solve?

Issue Number: close #42178 

Problem Summary:

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

**Comment by yiwen92** [comment]
@ran-huang PTAL

File: roadmap.md [hunk 1]
```diff
@@ -335,7 +335,7 @@ This roadmap brings you what's coming in the future, so you can see the new feat
           <br />
           <li>
             <b>Database encryption</b>
-            <br /><i>Support data-at-rest encryption for database files</i>
+            <br /><i>Encryption at rest with more granularity in table-level and column-level.</i>
```
Comment (ran-huang) [inline roadmap.md:338]:
```suggestion
            <br /><i>Encryption at rest with more granularity in table-level and column-level</i>
```

**Review by ran-huang — APPROVED** [review]
LGTM

**Review by dveeden — APPROVED** [review]

**Comment by yiwen92** [comment]
pls help to merge @bb7133 thx！

**Pull request merged**